<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningForm</class>
 <widget class="QWidget" name="WarningForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="RdBtn_RTK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>RTK NoLink 0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RdBtn_RemoteController">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>遥控器未连接</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RdBtn_PsiRTK">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>航向RTK NoLink 0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RdBtn_GPS">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>GPS NoLink 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
